src/pages/commits/BuildVariantAccordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariants: {
+    displayName: string;
+  }[];
+}
+export const BuildVariantAccordion: React.FC<Props> = ({ buildVariants }) => (
```

Comment:
nit. I think this should be renamed to something more representative of what it is. BuildVariantAccordion sounds like it should be a singular element when in reality this component represents a container for a bunch of buildVariantAccordions. Maybe BuildVariantAccordionContainer or something better?

Reply:
Yes good idea! Thanks!